src/fetch.js

```diff
@@ -41,6 +41,7 @@ function cozyFetchWithAuth (cozy, fullpath, options, credentials) {
     if (!client || !(token instanceof AccessToken)) {
       return res
     }
+    options.dont_retry = true
```

Comment:
Looks ok to me, I just think that the property should be camelCase, i.e. `dontRetry` instead of `dont_retry`.